Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apt-get update && apt-get install -y \
 	libpng-dev \
 	libxml2-dev \
 	msmtp \
-	mysql-client \
+	mariadb-client \
```

Comment:
Fixes a bug where the repo wouldn't work. See https://stackoverflow.com/questions/57048428/e-package-mysql-client-has-no-installation-candidate-in-php-fpm-image-build-u